docs/nx-cloud/reference/release-notes.md

```diff
@@ -1,5 +1,58 @@
 # Enterprise Release Notes
 
+### 2025.07
+
+##### Breaking Change
+
+This upgrade includes a breaking change to the `nx-cloud` cluster: instead of a message queue, the `nx-api` pod needs a valid Valkey (Redis) connection string.
+
+1. Install Valkey:
+   1. You can either use the Bitnami chart: https://github.com/bitnami/charts/tree/main/bitnami/valkey
+   2. Or for a simpler deployment, you can use the Valkey docker image directly: https://hub.docker.com/r/valkey/valkey/
+   3. Or you can install it as a system service: https://valkey.io/topics/installation/
+2. Upgrade to the latest Helm chart `0.16.3`
+3. Apply the below values
+
+```yaml
+enableMessageQueue: false
+
+nxApi:
+  # add these env vars to the nx-api
+  deployment:
+    env:
+      - name: VALKEY_CLIENT_PROVIDER
+        value: 'redisson'
+      - name: VALKEY_PASSWORD
+        valueFrom:
+          # remember to apply this secret to your cluster
+          secretKeyRef:
+            name: valkey-secrets
+            key: VALKEY_PASSWORD
+      - name: VALKEY_PORT
+        value: '6379'
+      - name: VALKEY_PRIMARY_ADDRESS
+        value: 'valkey'
+      - name: VALKEY_USE_SENTINEL
+        value: 'false'
```

Comment:
should we encourage using sentinel or just let people decide if that's something they care about when applying?